ethstore/cli/src/main.rs

```diff
@@ -145,7 +145,6 @@ impl fmt::Display for Error {
 }
 
 fn main() {
-	panic_hook::set_abort();
```

Comment:
Why is this removed?

Reply:
same question

Reply:
don't remember exactly, but i has some local issues with it. Restoring